tests/Integration/Database/EloquentBelongsToManyTest.php

```diff
@@ -48,7 +48,8 @@ protected function setUp(): void
 
         Schema::create('posts_tags', function (Blueprint $table) {
             $table->integer('post_id');
-            $table->string('tag_id');
+            $table->integer('tag_id')->default(0);
+            $table->string('tag_name')->default('')->nullable();
```

Comment:
Using an integer for the related custom key leads to errors in PostgreSQL. Since the purpose for the related test is a custom related string key we should also make sure the pivot column has the correct type. There for we need to add one explicitly.